macros/create_pipe.sql

```diff
@@ -53,6 +63,20 @@ copy into {{ schema_name }}.{{ table_name }} from (
     )
 {% endset %}
 
+{% set aws_ingest_copy_statement %}
+  copy into {{ schema_name }}.{{ table_name }}
+  from @{{ schema_name }}.{{ table_name }}_stage
+  file_format = (
+    type = '{{ file_type }}'
+    {% for key, value in format_type_options.items() %}
+      {{- key }} = {{ value }}
+    {% endfor -%}
+    )
+  {{ "match_by_column_name = '" ~ pipe.match_by_column_name ~ "'" if pipe.match_by_column_name }}
```

Comment:
match_by_column_name is part of the [copyOptions](https://docs.snowflake.com/en/sql-reference/sql/copy-into-table#copy-options-copyoptions) config for copy into. wondering if we could use that similar to the format_options is set up

Reply:
Would this require us to make `copy_into_options` as a new key in the Snowflake config, much like how we do with `extra_format_options`? Just thinking about doing this, but want to maintain the same behavior in which the provided options will override defaults.

Reply:
I think that's a great option to extend the functionality of what you're trying to achieve here.

We don't set any defaults for these copyOptions so it wouldn't extend functionality, but instead just set for each pipe.